### PR TITLE
Issue #2857560: Complete the coupon <-> promotion data model

### PR DESCRIPTION
**Pull request by vasike**

### CODE REVIEW

File: modules/promotion/commerce_promotion.post_update.php [hunk 1]
```diff
@@ -5,6 +5,7 @@
  * Post update functions for Promotion.
  */
 
+use Drupal\Core\Field\BaseFieldDefinition;
```
Comment (mglaman) [inline modules/promotion/commerce_promotion.post_update.php]:
```
----------------------------------------------------------------------
  8 | ERROR | [x] There must be one blank line after the last USE
    |       |     statement; 0 found;
    |       |     (PSR2.Namespaces.UseDeclaration.SpaceAfterLastUse)
```

Comment (vasike) [inline modules/promotion/commerce_promotion.post_update.php]: done. thanks

**Review by mglaman — APPROVED** [review]

**Comment by mglaman** [comment]
@bojanz this looks good to me, adds backreference and required helper methods. I'd like to get your gut feeling :)

**Comment by bojanz** [comment]
Committed in 4aaf1d7cbd2aec89e743f3faff82fb5da386408f.

**Pull request closed without merging**